### DISCUSSION TITLE
Missing index.js for v0.47.1

### DISCUSSION THREAD

**Discussion by beherap** [outline/outline#1550] [General] [answered]
Hello, I am having some trouble while trying to start the services in Step 5 of the documentation.

> Start the service with any daemon tools you prefer. Take PM2 for example, `NODE_ENV=production pm2 start index.js --name outline`

Looks like the `index.js` file is missing.

**Comment by tommoor** [comment, marked as answer]
Thanks, the documentation needs updating. You should run from the tag, not from master:
https://github.com/outline/outline/tree/v0.47.1

OR even better, the docker image:
`outlinewiki/outline:latest`

**Reply by beherap** [reply]
Thank you!!
